Zend/zend_exceptions.stub.php

```diff
@@ -119,3 +119,26 @@ class ArithmeticError extends Error
 class DivisionByZeroError extends ArithmeticError
 {
 }
+
+final class StackFrame
+{
+    public static function getTrace(int $options = DEBUG_BACKTRACE_PROVIDE_OBJECT, int $limit = 0): array {}
+
+    public function getFile(): string {}
+
+    public function getLine(): int {}
+
+    public function getFunction(): ?string {}
+
+    public function getClass(): ?string {}
+
+    public function getObject(): ?object {}
+
+    public function getObjectClass(): ?string {}
+
+    public function getClosure(): ?\Closure {}
+
+    public function getType(): ?string {}
+
+    public function getArgs(): array {}
+}
```

Comment:
It isn't enough to just declare that it implements ArrayAccess. You also have to declare the stubs and support calling offsetGet directly

```
php > echo (new ReflectionClass('StackFrame'))->offsetGet('file');

Warning: Uncaught Error: Call to undefined method ReflectionClass::offsetGet() in php shell code:1
Stack trace:
#0 {main}
  thrown in php shell code on line 1
php > var_export((new ReflectionClass('StackFrame'))->isAbstract());
true
```